jwt.go

```diff
@@ -19,13 +19,193 @@ var TimeFunc = time.Now
 // Header of the token (such as `kid`) to identify which key to use.
 type Keyfunc func(*Token) (interface{}, error)
 
+// For a type to be a Claims object, it must just have a Valid method that determines
+// if the token is invalid for any supported reason
+type Claimer interface {
```

Comment:
What would the method Claim return / be intended to do?

I could say it was due to the concept that a interface should describe "actions", and make sense suffixed with "er", "or", etc...

Honestly, though, it was because it was shorter for later typing. :)
When you're inspecting the values, you still need to do.. 

```go
claim := token.Claims.(*jwt.Claims) 
```